protos/protos.d.ts

```diff
@@ -8,3205 +8,6 @@ export namespace google {
         /** Namespace translation. */
         namespace translation {
 
-            /** Namespace v3. */
-            namespace v3 {
```

Comment:
@bcoe IIRC v3 was added manually? Should we just be ignoring these PRs in the mean time?